bin/kafka-run-class.sh

```diff
@@ -292,7 +292,8 @@ if [ "x$GC_LOG_ENABLED" = "xtrue" ]; then
   # 10.0.1 -> java version "10.0.1" 2018-04-17
   # We need to match to the end of the line to prevent sed from printing the characters that do not match
   JAVA_MAJOR_VERSION=$($JAVA -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
-  if [[ "$JAVA_MAJOR_VERSION" -ge "9" ]] ; then
+  JAVA_VM_ISJ9=$($JAVA -version 2>&1|grep J9)
```

Comment:
Can we rename the variable so it's clear we're talking about OpenJ9 and not Java 9?
Can we also grep for `openj9` instead of simply `J9` so there's no chance it matches something else

Reply:
@mimaison I agree with your proposal, I will send soon a new commit taking it into account.

Reply:
+1 for `openj9`. :smiley:

Reply:
Done pushing the requested changes!
Any more feedback on this PR ?